src/core_plugins/kibana/public/dashboard/dashboard_state.js

```diff
@@ -122,7 +122,7 @@ export class DashboardState {
     // in place in order for it to affect the visualizations.
     this.stateDefaults.query = this.lastSavedDashboardFilters.query;
     // Need to make a copy to ensure they are not overwritten.
-    this.stateDefaults.filters = Object.assign(new Array(), this.getLastSavedFilterBars());
+    this.stateDefaults.filters = this.getLastSavedFilterBars().slice();
```

Comment:
Our style guide wants this to be `[...this.getLastSavedFilterBars()]` instead of using `.slice()`